processing/sensorhub-process-opencv/build.gradle

```diff
@@ -17,9 +17,14 @@ dependencies {
 osgi {
   manifest {
     attributes('Bundle-Vendor': 'Sensia Software LLC')
-    attributes('Bundle-Activator': 'org.sensorhub.impl.process.opencv')
+    attributes('Bundle-Activator': 'org.sensorhub.impl.process.opencv.Activator')
     attributes('-fixupmessages': 'Classes found in the wrong directory; is:=ignore')
-    attributes('-exportcontents': 'org.bytedeco.opencv.*, org.opencv.*')
+    attributes('-exportcontents': 'org.bytedeco.opencv, org.opencv,org.sensorhub.impl.process.opencv')
+    attributes('Import-Package': '''
+      !android.*, !java.*, !javax.*, !org.bytedeco.cpython.global.*, !org.bytedeco.javacpp.*,
+      !org.bytedeco.numpy.global.*,
+      org.osgi.service.repository;resolution:=optional,
```

Comment:
Don't think we need to import "org.osgi.service.repository;resolution:=optional"

Reply:
Ok, let's remove it everywhere then; I don't think any modules in addons need it for now